test/test.js

```diff
@@ -27,57 +29,35 @@ test('package', function (t) {
 })
 
 test('throw if missing required options', function (t) {
-  t.throws(function () {
-    new Client() // eslint-disable-line no-new
-  })
-  t.end()
-})
-
-test('throw if only userAgent is provided', function (t) {
-  t.throws(function () {
-    new Client({userAgent: 'foo'}) // eslint-disable-line no-new
-  })
-  t.end()
-})
-
-test('throw if only meta is provided', function (t) {
-  t.throws(function () {
-    new Client({meta: onmeta}) // eslint-disable-line no-new
-  })
-  t.end()
-})
-
-test('only userAgent and meta should be required', function (t) {
-  t.doesNotThrow(function () {
-    new Client({ // eslint-disable-line no-new
-      userAgent: 'foo',
-      meta: onmeta
-    })
-  })
+  t.throws(() => new Client())
+  t.throws(() => new Client({agentName: 'foo'}))
+  t.throws(() => new Client({agentVersion: 'foo'}))
+  t.throws(() => new Client({serviceName: 'foo'}))
+  t.throws(() => new Client({userAgent: 'foo'}))
+  t.throws(() => new Client({agentName: 'foo', agentVersion: 'foo', serviceName: 'foo'}))
+  t.throws(() => new Client({agentName: 'foo', agentVersion: 'foo', userAgent: 'foo'}))
+  t.throws(() => new Client({agentName: 'foo', serviceName: 'foo', userAgent: 'foo'}))
+  t.throws(() => new Client({agentVersion: 'foo', serviceName: 'foo', userAgent: 'foo'}))
+  t.doesNotThrow(() => new Client({agentName: 'foo', agentVersion: 'foo', serviceName: 'foo', userAgent: 'foo'}))
```

Comment:
Can you provide messages for these assertions, so it's clear _what_ these are testing?

Reply:
Fixed